### DISCUSSION TITLE
Frames are hidden as three dots in call tree view

### DISCUSSION THREAD

**Discussion by xxks-kkk** [async-profiler/async-profiler#1052] [Q&A] [answered]
Hello,

As shown below, in the call tree view, I see there are 40 frames but there is no way I can see what exactly those are as indicated by the three dots. Is there anyway I can see those as well? Thank you!

<img width="218" alt="image" src="https://github.com/user-attachments/assets/69fb37c5-ef6b-4b54-af62-881bf8dd0294">

**Comment by apangin** [comment, marked as answer]
By default, small frames are pruned from the tree view to avoid generating abnormally huge .html files.
This is controlled by `--minwidth` option.
If you add `--minwidth 0.001` option, generated tree will include the smallest frames.
